packages/ui/molecules/Hero/Hero.twig

```diff
@@ -0,0 +1,143 @@
+{#
+/**
+ * @file
+ * Hero component.
+ *
+ * @param string $surtitle
+ * @param string $title
+ * @param string $subtitle
+ * @param array $ctas[]
+ * @param string $cta_url
+ * @param string $cta_label
+ *
+ * @param array                                     $attr
+ *   customize the root element attributes.
+ * @param array<Figure> $image
+ *   Image to display. See Figure.twig for the composition.
+ * @param array                                     $image_attr
+ *   customize the image element attributes.
+ * @param array                                     $content_attr
+ *   customize the content classes.
+ * @param 'left'|'center'|'right'                   $content_position
+ *   define the content position.
+ * @param array                                     $surtitle_tag
+ *   customize the tag of the surtitle element
+ * @param array                                     $surtitle_attr
+ *   customize the surtitle element attributes.
+ * @param array                                     $title_tag
+ *   customize the tag of the title element
+ * @param array                                     $title_attr
+ *   customize the title element attributes.
+ * @param array                                     $description_tag
+ *   customize the tag of the description element
+ * @param array                                     $description_attr
+ *   customize the description element attributes.
+ * @param array                                     $cta_attr
```

Comment:
**nitpick:** Rename `cta_attr` to `ctas_attr`

This variable can be used to customize the attributes of the element wrapping the CTAs of the hero.